test/functionalTest/cases/1678_geosubscriptions/geosubscriptions.test

```diff
@@ -42,7 +42,7 @@ accumulatorStart --pretty-print
 # 09. With subscription in place, create entity E3 inside area A (notif)
 # 10. Update subscription using a micro-area centered at E4
 # 11. With subscription in place, create entity E4 outside area A (notif)
-# 12. Dump acumulator (4 notifs: E1, E2, E1, E1, E3, E4)
+# 12. Dump acumulator (6 notifs: E1, E2, E1, E1, E3, E4)
```

Comment:
Small typo fix in this .test. Not related with the PR topic itself, but spotted while I was debugging cases.

NTC (informative)